services/backend/src/core/search/util/anyMentorSkillsSearch.js

```diff
@@ -0,0 +1,13 @@
+async function anyMentorshipSkillSearch(profiles) {
+  const skillProf = [];
+
+  profiles.forEach((profile) => {
+    if (profile.mentorshipSkills && profile.mentorshipSkills.length) {

```

Comment:
I think array.length is only thruthy when its equal to 1
```suggestion
    if (profile.mentorshipSkills && profile.mentorshipSkills.length > 0) {
```

Reply:
All numbers greater than 0 are truthy, but that does look more clear